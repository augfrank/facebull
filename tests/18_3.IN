M0   C0   C3  2532
M1   C0   C8  2302
M2   C0   C11  2475
M3   C0   C13  2962
M4   C1   C2  2549
M5   C1   C4  2909
M6   C1   C6  2160
M7   C1   C7  2358
M8   C1   C8  2227
M9   C1   C10  2727
M10   C1   C15  2235
M11   C1   C17  2062
M12   C2   C3  2431
M13   C2   C4  2076
M14   C2   C5  2068
M15   C2   C11  2509
M16   C2   C12  2155
M17   C2   C13  2999
M18   C2   C16  2821
M19   C3   C2  2492
M20   C3   C7  2069
M21   C3   C8  2468
M22   C3   C15  2625
M23   C3   C17  2954
M24   C4   C5  2396
M25   C4   C9  2632
M26   C4   C10  2771
M27   C4   C13  2350
M28   C4   C15  2755
M29   C4   C16  2257
M30   C5   C1  2330
M31   C5   C6  2296
M32   C5   C12  2212
M33   C5   C14  2115
M34   C5   C16  2987
M35   C6   C3  2154
M36   C6   C10  2538
M37   C6   C11  2602
M38   C6   C12  2727
M39   C6   C14  2284
M40   C6   C16  2899
M41   C7   C1  2733
M42   C7   C2  2718
M43   C7   C4  2256
M44   C7   C9  2314
M45   C7   C10  2639
M46   C7   C11  2556
M47   C7   C13  2354
M48   C7   C14  2314
M49   C7   C15  2483
M50   C8   C1  2586
M51   C8   C4  2046
M52   C8   C5  2567
M53   C8   C11  2194
M54   C8   C13  2454
M55   C8   C15  2387
M56   C9   C0  2875
M57   C9   C1  2879
M58   C9   C2  2268
M59   C9   C4  2900
M60   C9   C10  2742
M61   C9   C12  2842
M62   C9   C14  2372
M63   C10   C3  2842
M64   C10   C11  2192
M65   C10   C13  2542
M66   C10   C14  2210
M67   C11   C1  2572
M68   C11   C4  2571
M69   C11   C6  2094
M70   C11   C12  2280
M71   C11   C14  2929
M72   C11   C16  2082
M73   C12   C0  2513
M74   C12   C1  2440
M75   C12   C3  2497
M76   C12   C7  2862
M77   C12   C9  2020
M78   C12   C10  2912
M79   C13   C2  2143
M80   C13   C4  2948
M81   C13   C7  2084
M82   C13   C8  2366
M83   C13   C9  2377
M84   C13   C10  2162
M85   C13   C15  2655
M86   C13   C16  2682
M87   C13   C17  2394
M88   C14   C0  2431
M89   C14   C3  2433
M90   C14   C5  2538
M91   C14   C6  2485
M92   C14   C7  2272
M93   C14   C15  2721
M94   C14   C16  2097
M95   C14   C17  2490
M96   C15   C8  2985
M97   C15   C13  2293
M98   C15   C14  2523
M99   C15   C16  2240
M100   C15   C17  2173
M101   C16   C2  2673
M102   C16   C4  2429
M103   C16   C7  2849
M104   C16   C9  2124
M105   C16   C11  2072
M106   C16   C12  2884
M107   C16   C15  2828
M108   C16   C17  2440
M109   C17   C2  2419
M110   C17   C4  2695
M111   C17   C7  2918
M112   C17   C9  2598
M113   C17   C14  2311
