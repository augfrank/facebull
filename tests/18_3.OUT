40008
1 11 14 20 27 32 35 49 51 58 64 69 74 84 88 99 104 113
